openms-wiki/contributors-quickstart-guide.md

```diff
@@ -0,0 +1,61 @@
+# Contribute to OpenMS
+
+If you would like to contribute to OpenMS:
+
+* Familiarise yourself with our [online documentation](https://abibuilder.informatik.uni-tuebingen.de/archive/openms/Documentation/release/latest/html/index.html).
+
+* Learn how to [build OpenMS](https://github.com/OpenMS/OpenMS/wiki/Building-OpenMS).
```

Comment:
let's have a numbered list, instead of bullet points.

Reply:
let's replace
> * Learn how to [build OpenMS](https://github.com/OpenMS/OpenMS/wiki/Building-OpenMS).

2. [Build OpenMS]() from source.

you might want to correct the building instructions link to API ref.